platform-hub-web/src/app/app.routes.js

```diff
@@ -771,15 +771,15 @@ export const appRoutes = function ($stateProvider, $urlRouterProvider, $location
         },
         data: {
           authenticate: true,
-          rolePermitted: 'admin'
+          rolesPermitted: ['admin', 'limited_admin']
         }
       })
       .state('costs-reports.new', {
         url: '/new',
         component: CostsReportsForm,
         data: {
           authenticate: true,
-          rolePermitted: 'admin'
+          rolesPermitted: ['admin']
```

Comment:
Would we also want to give `limited_admin` people the ability to create new cost reports?

Reply:
I don't believe we'll be doing that at the moment – I think it's purely for consumption at this stage (though it's easy to add access for it later, if needed).